packages/genesys-spark-components/src/components/stable/gux-modal/gux-modal.scss

```diff
@@ -93,6 +93,17 @@ slot[name='start-align-buttons']::slotted(:not(button, gux-button-slot)) {
           display: none;
         }
       }
+
+      footer {
+        padding-right: var(--gse-ui-modal-padding);
+        padding-bottom: var(--gse-ui-modal-padding);
+        padding-left: var(--gse-ui-modal-padding);
+        margin-top: var(--gse-ui-modal-gap);
+
+        &.gux-no-buttons {
+          display: none;
```

Comment:
Is this needed here ?

Reply:
Yes, there is margin & padding that added unnecessary space when there are no buttons, see the 'Only Content' example.

Reply:
Ahh okay I was confused because you nesting the class inside the `footer` and are applying it like `footer class="gux-no-buttons"></footer>` but in the tsx file I cannot find where you are rendering it the function just looks like 

```
 private renderFooter(): JSX.Element {
    if (hasSlot(this.root, 'footer')) {
      return (
        <footer>
          <slot name="footer" />
        </footer>
      ) as JSX.Element;
    } else {
      return this.renderButtonFooter();
    }
  }
```

Reply:
Sorry @gavin-everett-genesys, yes you are correct, good spot. 
I have 2 `.gux-no-buttons`, don't need the one on the footer (copy & paste probably).